### PR TITLE
v0.15.0 Fixes / Enhancements

### PR DESCRIPTION
**Pull request by huseyinbabal**
## Description

Changes proposed in this pull request:

- Introduced support for the text filter that has space in it
- Error message template has changed for filter execution response

## Testing
Same as [here](https://github.com/kubeshop/botkube/pull/805)

## Proof
Screen Shot 2022-10-20 at 17 51 31

### CODE REVIEW

File: helm/botkube/Chart.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 name: botkube
 home: https://botkube.io
-version: v9.99.9-dev
-appVersion: v9.99.9-dev
+version: v0.15.0-rc.1
+appVersion: v0.15.0-rc.1
```
Comment (pkosiec) [inline helm/botkube/Chart.yaml:5]: I think all the versions shouldn't change as the release pipeline will override all of the stuff. Correct?

**Review by pkosiec — APPROVED** [review]
Works OK, but please see my one comment before merge

File: helm/botkube/Chart.yaml (hunk 1 above)
Comment (huseyinbabal) [reply helm/botkube/Chart.yaml]: they will be overwritten once release is finalized

**Comment by ezodude** [comment]
@huseyinbabal found a very minor issue, that perhaps can be included in this fix?

[See second big in  update v0.15.0 bugs](https://github.com/kubeshop/botkube/issues/825#issuecomment-1285456828) - can create a separate PR if this doesn't make sense.

**Comment by huseyinbabal** [comment]
let's include it here :) it will be easy to cherry pick

**Comment by ezodude** [comment]
Appreciated.

**Comment by huseyinbabal** [comment]
@ezodude as agreed, the changes are in remote, and here is the final working state of the error response
Screen Shot 2022-10-21 at 11 04 38

**Review by ezodude — APPROVED** [review]
👍

**Pull request merged**